### ISSUE TITLE
/proc/fs/nfsd being flagged as an NFS share

### ISSUE THREAD

**Issue by brianjmurrell** [oamg/leapp-repository#807]
https://github.com/oamg/leapp-repository/blob/f3e4056c70a20b35c7718f3838f7ecd432e757b5/repos/system_upgrade/common/actors/checknfs/actor.py#L35 is (incorrectly, IMHO) flagging `nfsd /proc/fs/nfsd nfsd rw,relatime 0 0` as an NFS share and inhibiting upgrade:

```
Risk Factor: high (inhibitor)
Title: Use of NFS detected. Upgrade can't proceed
Summary: NFS is currently not supported by the inplace upgrade.
We have found NFS usage at the following locations:
- Currently mounted NFS share:
nfsd /proc/fs/nfsd nfsd rw,relatime

Remediation: [hint] Disable NFS temporarily for the upgrade if possible.
Key: 9881b25faceeeaa7a6478bcdac29afd7f6baaaed
```
As an aside, the line above showing the NFS mount that is being flagged is added by #806.

**Comment by pirat89** [comment]
Hi @brianjmurrell. thank you for the report and improved logging. We are already aware about this issue and we are keeping it for now as good simple issue for our newcomer (in Feb). It's definitely one of things we want to deliver in the next release. However in case of need, PRs are always welcomed, we have more good first issues in our list :)

**Comment by brianjmurrell** [comment]
Probably looking for `in ['nfs3', 'nfs4']` is a decent solution.  More future proof perhaps might be an RE match against `/nfs[3-9]/` although that is a bit presumptive.  :-)

**Comment by pirat89** [comment]
@brianjmurrell  Exactly something like that is expected fix :) In case of the list, I am not sure whether it's possible to have a type like `nfs4.2` or whether specification of particular "minor" version is possible  just through additional options like `vers=4.2`. Anyway, definitely we will check that NFS severs will not be automatically blocked for upgrades.

**Comment by pirat89** [comment]
For the tracking when this is fixed in RHEL:
* https://bugzilla.redhat.com/show_bug.cgi?id=2036069

The known workaround in the meanwhile
```
# systemctl stop /proc/fs/nfsd
```

**Comment by pirat89** [comment]
This has been already fixed in release v0.17.0 - PR #888
